src/components/tabset/tabset.component.spec.ts

```diff
@@ -68,6 +72,7 @@ describe('Tabset Component', () => {
         fixture = TestBed.createComponent(TabsetTestComponent);
         component = fixture.componentInstance;
         nativeElement = fixture.nativeElement;
+        tabset = fixture.debugElement.query(By.directive(TabsetComponent)).componentInstance;
```

Comment:
This is fine, I just wanted to note that a possibly easier/ more maintainable way of accomplishing this is to add something like
```ts
@ViewChild('tabset') public tabset: TabsetComponent;
```
to the TabsetTestComponent. This is how one would normally access a component's public functions.

Reply:
Thanks for this Donna, was trying to figure out how to do it this way and it was Ashley who helped me with the other solution but this is helpful to know for future reference.